autoload/airline/extensions/branch.vim

```diff
@@ -3,11 +3,14 @@
 
 scriptencoding utf-8
 
+" TODO: Checking for autoload functions is a bad idea. exists checks don't
+" work for autoload functions unless they happen to be loaded before us.
 let s:has_fugitive = exists('*fugitive#head')
 let s:has_lawrencium = exists('*lawrencium#statusline')
 let s:has_vcscommand = get(g:, 'airline#extensions#branch#use_vcscommand', 0) && exists('*VCSCommandGetStatusLine')
+let s:has_custom_scm = exists('airline#extensions#branch#custom_head')
```

Comment:
do not use a custom `s:var` here. That way, users setting the `g:airline#extensions#branch#custom_head` cannot change it later, because the `s:var` is fixed and won't change.